examples/foreach/tests/foreach_test.bal

```diff
@@ -10,21 +10,44 @@ int counter = 0;
     functionName: "println"
 }
 public function mockPrint(any... s) {
-    outputs[counter] = s[0];
+    string printString = "";
+    foreach var val in s {
+        printString += string.convert(val);
+    }
+    outputs[counter] = printString;
     counter += 1;
 }
 
 @test:Config
 function testFunc() {
     // Invoking the main function
     main();
+    test:assertEquals(outputs[0], "Iterating over a string array :");
     test:assertEquals(outputs[1], "fruit: apple");
     test:assertEquals(outputs[2], "fruit: banana");
     test:assertEquals(outputs[3], "fruit: cherry");
 
+    test:assertEquals(outputs[4], "\nIterating over a map :");
     test:assertEquals(outputs[5], "letter: a, word: apple");
     test:assertEquals(outputs[6], "letter: b, word: banana");
     test:assertEquals(outputs[7], "letter: c, word: cherry");
 
-    //TODO : Add the rest of the assertions
+    test:assertEquals(outputs[8], "\nIterating over a json object :");
+    test:assertEquals(outputs[9], "string value: apple");
+    test:assertEquals(outputs[10], "json array value: [\"red\", \"green\"]");
+    test:assertEquals(outputs[11], "int value: 5");
+
+    test:assertEquals(outputs[12], "\nIterating over a json array :");
```

Comment:
```suggestion
    test:assertEquals(outputs[12], "\nIterating a JSON array :");
```

Reply:
IMO, we need "over" here

Reply:
Same for integer ranges too.

Reply:
We iterate something, not over something. Therefore, should say "iterate xyz...." etc.